[Bulk] Broken Link fixes (#878)
MicrosoftDocs/biztalk-docs: biztalk/core/test-scenarios-for-measuring-mst-of-dta-tracking.md

   For example, we have provided all of the hardware, configuration, and solution architecture for the sample scenario in this topic. Using that, and the type of tracking we had turned on (default + 10 properties + 10KB message bodies) and the MST we achieved (20 msgs/sec) with a 24 hour DTA data retention window, compare that to your setup and requirements.

>    You should at least be able to estimate if the MST for your solution will be higher or lower than what we achieved. Various technical case studies may also be available to which you can compare your system. For examples, see the case studies available on the BizTalk Server Developer Center.

Reviewer: No more BizTalk Server Developer Center:

```suggestion
   You should at least be able to estimate if the MST for your solution will be higher or lower than what we achieved. Various technical case studies might also be available to which you can compare your system.
```
